translator/GCP_cloud_function/README.md

```diff
@@ -1,9 +1,10 @@
 # iCone translator Cloud Function
 
-A system was created in google cloud platform to automatically translate iCone data to WZDx message.This system consits of two buckets and a cloud function.
-When a file is uploaded to the first bucket, the cloud function automatically downloads the file, translates to a WZDx message and uploads it to the second bucket.
+A system was created in google cloud platform to automatically translate iCone data to WZDx message. This system consists of two pubsub topics and a cloud function.
```

Comment:
I'm only seeing the single pubsub topic that we are sending data to at this point. Looking at `main.py` it appears that the current logic is as follows:
1. hello_pubsub triggered (not sure how)
2. call out to FTP site and pull file
3. parse file to xml
4. translate xml to wzdx object
5. call validation function
6. publish wzdx object to pubsub

In general, we shouldn't be publishing to a pubsub topic from a scheduler just to kick off a cloud function. We can set the scheduler to call the function directly.

Reply:
The second pub/sub topic is for triggering the function.we didn't know ,we could trigger it directly from cloud scheduler ,we will remove the second pub/sub topic in next sprint.

Reply:
@abinash-konersman can you write up a story for this and throw a link to it here so we have it documented.

Reply:
After working with Dhivahari a bit on the pubsub/scheduler stuff you are correct here. It appears the best way to get the scheduler to talk to the cloud function is sending a message through pubsub (which just seems very strange, but it's what the documentation says to do!). No need to write up the story mentioned above. Let's do get the README in line with what the code is actually doing though.